Framework/DataHandling/src/SaveDiffCal.cpp

```diff
@@ -71,9 +72,7 @@ std::map<std::string, std::string> SaveDiffCal::validateInputs() {
   std::map<std::string, std::string> result;
 
   ITableWorkspace_const_sptr calibrationWS = getProperty("CalibrationWorkspace");
-  if (!bool(calibrationWS)) {
-    result["CalibrationWorkspace"] = "Cannot save empty table";
-  } else {
+  if (bool(calibrationWS)) {
```

Comment:
Is there a reason we use `if bool(calibrationWS)` as opposed to straight `if(calibrationWS)`? I am guilty of that myself, having done a similar thing elsewhere in this file.

Reply:
I remember doing the `bool` vs not and don't remember why. I feel like there was a reason at one point.